model/fn-execution/build.gradle

```diff
@@ -17,7 +17,10 @@
  */
 
 plugins { id 'org.apache.beam.module' }
-applyPortabilityNature(shadowJarValidationExcludes: ["org/apache/beam/model/fnexecution/v1/**"])
+applyPortabilityNature(
+    automaticModuleName: 'beam.model.fn.execution',
```

Comment:
wondering if for this and the other modules it would be better to have the fully qualified name: `org.apache.beam.model.fn.execution`, `org.apache.beam....`. It could be long but it seems to be the convention, What do others think?
I assume that some project can choose the beam project and have some clash with us, however in principle nobody will choose the fully qualified Apache path.

Reply:
You are right - I should change it as you suggest. There are strong arguments to do so: https://blog.joda.org/2017/04/java-se-9-jpms-module-naming.html